src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,352 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  RefreshControl,
+  ScrollView,
+  SectionList,
+  SectionListProps,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native'
+import Animated from 'react-native-reanimated'
+import { SafeAreaView, useSafeAreaInsets } from 'react-native-safe-area-context'
+import { useDispatch, useSelector } from 'react-redux'
+import { DappExplorerEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Touchable from 'src/components/Touchable'
+import {
+  CategoryWithDapps,
+  dappFavoritesEnabledSelector,
+  dappsCategoriesAlphabeticalSelector,
+  dappsListErrorSelector,
+  dappsListLoadingSelector,
+  dappsListSelector,
+  dappsMinimalDisclaimerEnabledSelector,
+  favoriteDappIdsSelector,
+} from 'src/dapps/selectors'
+import { fetchDappsList } from 'src/dapps/slice'
+import { DappFilter, DappSection, DappV1, DappV2 } from 'src/dapps/types'
+import DappCard from 'src/dappsExplorer/DappCard'
+import FavoriteDappsSection from 'src/dappsExplorer/FavoriteDappsSection'
+import { NoResults } from 'src/dappsExplorer/NoResults'
+import useDappFavoritedToast from 'src/dappsExplorer/useDappFavoritedToast'
+import useDappInfoBottomSheet from 'src/dappsExplorer/useDappInfoBottomSheet'
+import useOpenDapp from 'src/dappsExplorer/useOpenDapp'
+import Help from 'src/icons/Help'
+import DrawerTopBar from 'src/navigator/DrawerTopBar'
+import { TopBarIconButton } from 'src/navigator/TopBarButton'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+const AnimatedSectionList =
+  Animated.createAnimatedComponent<SectionListProps<DappV1 | DappV2, SectionData>>(SectionList)
```

Comment:
shouldn't this component only reference `DappV2`? I think you can remove all references to `DappV1`, and the correct type for `category` in `SectionData` should be string instead of `CategoryWithDapps`?

Reply:
Fixed in 5e4e9be0f975baa845bf43846989f61cb2a4ef99!